common/CMakeLists.txt

```diff
@@ -15,15 +15,21 @@ include(GNUInstallDirs)
 # Default values
 set(BEEROCKS_BRIDGE_IFACE "br-lan")
 set(BEEROCKS_PLATFORM_INIT "false")
-set(BEEROCKS_INSTALL_PATH "/opt/prplmesh")
+set(BEEROCKS_INSTALL_PATH "/opt/prplMesh")
 set(BEEROCKS_TMP_PATH "/tmp/beerocks")
 set(BEEROCKS_WLAN1_IFACE "wlan0")
 set(BEEROCKS_WLAN2_IFACE "wlan2")
+set(BEEROCKS_BH_WIRE_IFACE "eth1")
 
 # Platform specific overrides
-if (TARGET_PLATFORM STREQUAL "ugw")
-    add_definitions(-DBEEROCKS_UGW)
-    set(BEEROCKS_BH_WIRE_IFACE "eth1")
+if (TARGET_PLATFORM STREQUAL "openwrt")
+    # UGW
+    if (TARGET_PLATFORM_TYPE STREQUAL "ugw")
+        add_definitions(-DBEEROCKS_UGW)
+    # Turris Omnia
+    elseif (TARGET_PLATFORM_TYPE STREQUAL "turris-omnia")
+        set(BEEROCKS_WLAN2_IFACE "wlan1")
```

Comment:
I think its better if we require turris omnia platforms to rename the wlan1 to wlan2 then have it hardcoded in our build system.

Reply:
??? Why would you want to rename wlan1 to wlan2? I think it's better to require whatever crazy platform that has wlan0 and wlan2 to rename wlan2 to wlan1 :-)

Reply:
Perhaps I'm biased towards Intel, but from what can remember we are not the only ones who mark STA interfaces with odd indexes. So wlan0,2,4 - APs, wlan1,3,5 - STAs.
Anyway, in most prplmesh supported platforms we will have this naming convention - UGW, RDKB, prplWrt - since they are Intel based.
I think the best solution is to just add it to the platform DB and never depend on the interface name. But this, of course, can be done in a separate PR, so (nitpick).

Reply:
> Perhaps I'm biased towards Intel, but from what can remember we are not the only ones who mark STA interfaces with odd indexes. So wlan0,2,4 - APs, wlan1,3,5 - STAs.

You're biased :-) That is *definitely* not how things should be done. If you have three radios, the first interface (=BSS) on each of them (STA or AP, doesn't matter) will be called wlan0, wlan1, wlan2. If you add a second interface, it will be called wlan0.1. In fact, the names are just names, hostapd and wpa_supplicant explicitly construct names like that. (wlan0 itself is a bit special because that name is constructed by the kernel, because the kernel will automatically add an interface to the phy when the driver is loaded, and that name is wlanN (typically, it actually depends on the driver).

Broadcom at least also follows this scheme, though the interfaces are called wl0, wl0.1 there.

I do suppose wlan0 and wlan1 are both on phy0 in the intel case, right?

> I think the best solution is to just add it to the platform DB and never depend on the interface name. But this, of course, can be done in a separate PR, so (nitpick).

Well, actually, it is only used to generate the config files. I think it's a bit extreme to have build options that are *only* used to generate a default config file...